pytripgui/canvas_vc/objects/ctx.py

```diff
@@ -22,6 +22,8 @@ def prepare_data_to_plot(self):
 
     def _set_aspect(self):
         if self.projection_selector.plane == "Transversal":
-            self.aspect = 1.0
-        else:
-            self.aspect = self.cube.slice_distance / self.cube.pixel_size
+            self.aspect = self.cube.dimx/self.cube.dimy
+        elif self.projection_selector.plane == "Sagittal":
```

Comment:
Please add more comments explaining:
  - what is aspect in general, is it `vertical / horizontal` or `horizontal / vertical` ?
For each of the views (transversal, sagittal, coronal) write down explicitly in a comment message what is on vertical axis and what on horizontal (X, Y or Z ?).

This information could be also added as the code (assuming some convention):
```
if  transveral:
   vertical_size = dimx * pixel_size
   horizontal_size = dimy * pixel size
(...)
aspect = vertical_size / horizontal_size
```

Reply:
Done